<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1280</width>
                <height>840</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>0</width>
                <height>0</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>16777215</width>
                <height>16777215</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                </sizepolicy>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>10</y>
                        <width>1251</width>
                        <height>761</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <property name="sizeConstraint">
                        <enum>QLayout::SetDefaultConstraint</enum>
                    </property>
                    <property name="leftMargin">
                        <number>5</number>
                    </property>
                    <property name="topMargin">
                        <number>5</number>
                    </property>
                    <property name="rightMargin">
                        <number>5</number>
                    </property>
                    <property name="bottomMargin">
                        <number>5</number>
                    </property>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                            <property name="spacing">
                                <number>1</number>
                            </property>
                            <item>
                                <widget class="Line" name="line">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="pushButton_close">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>20</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>&gt;</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QScrollArea" name="scrollArea">
                                    <property name="widgetResizable">
                                        <bool>true</bool>
                                    </property>
                                    <widget class="QWidget" name="scrollAreaWidgetContents">
                                        <property name="geometry">
                                            <rect>
                                                <x>0</x>
                                                <y>0</y>
                                                <width>1020</width>
                                                <height>717</height>
                                            </rect>
                                        </property>
                                    </widget>
                                </widget>
                            </item>
                            <item>
                                <widget class="Line" name="line_2">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QListWidget" name="listWidget">
                                    <property name="maximumSize">
                                        <size>
                                            <width>200</width>
                                            <height>16777215</height>
                                        </size>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1280</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="nativeMenuBar">
                <bool>true</bool>
            </property>
            <widget class="QMenu" name="menu_file">
                <property name="tearOffEnabled">
                    <bool>false</bool>
                </property>
                <property name="title">
                    <string>Файл</string>
                </property>
                <property name="toolTipsVisible">
                    <bool>false</bool>
                </property>
                <addaction name="action_open"/>
                <addaction name="separator"/>
                <addaction name="action_exit"/>
            </widget>
            <widget class="QMenu" name="menu_reference">
                <property name="title">
                    <string>Справка</string>
                </property>
                <addaction name="action_reference"/>
                <addaction name="separator"/>
                <addaction name="action_about_program"/>
            </widget>
            <addaction name="menu_file"/>
            <addaction name="menu_edit"/>
            <addaction name="menu_view"/>
            <addaction name="menu_scheme"/>
            <addaction name="menu_service"/>
            <addaction name="menu_reference"/>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <action name="action_create">
            <property name="text">
                <string>Создать</string>
            </property>
        </action>
        <action name="action_open">
            <property name="text">
                <string>Открыть</string>
            </property>
        </action>
        <action name="action_save">
            <property name="text">
                <string>Сохранить</string>
            </property>
        </action>
        <action name="action_exit">
            <property name="text">
                <string>Выход</string>
            </property>
        </action>
        <action name="action_cancel">
            <property name="text">
                <string>Отменить</string>
            </property>
        </action>
        <action name="action_repeat">
            <property name="text">
                <string>Повторить</string>
            </property>
        </action>
        <action name="action_cut">
            <property name="text">
                <string>Вырезать</string>
            </property>
        </action>
        <action name="action_copy">
            <property name="text">
                <string>Копировать</string>
            </property>
        </action>
        <action name="action_paste">
            <property name="text">
                <string>Вставить</string>
            </property>
        </action>
        <action name="action_paste_here">
            <property name="text">
                <string>Вставить здесь</string>
            </property>
        </action>
        <action name="action_duplicate">
            <property name="text">
                <string>Дублировать</string>
            </property>
        </action>
        <action name="action_find">
            <property name="text">
                <string>Найти/Заменить...</string>
            </property>
        </action>
        <action name="action_select_all">
            <property name="text">
                <string>Выделить все</string>
            </property>
        </action>
        <action name="action_align_left">
            <property name="text">
                <string>По левому краю</string>
            </property>
        </action>
        <action name="action_align_right">
            <property name="text">
                <string>По правому краю</string>
            </property>
        </action>
        <action name="action_align_horizontal_center">
            <property name="text">
                <string>По середине</string>
            </property>
        </action>
        <action name="action_align_up">
            <property name="text">
                <string>По верхнему краю</string>
            </property>
        </action>
        <action name="action_align_down">
            <property name="text">
                <string>По нижнему краю</string>
            </property>
        </action>
        <action name="action_align_vertical_center">
            <property name="text">
                <string>По центру</string>
            </property>
        </action>
        <action name="action_align_by_widest">
            <property name="text">
                <string>По самому широкому</string>
            </property>
        </action>
        <action name="action_align_by_tighest">
            <property name="text">
                <string>По самому узкому</string>
            </property>
        </action>
        <action name="action_tallest">
            <property name="text">
                <string>По самому высокому</string>
            </property>
        </action>
        <action name="action_lowest">
            <property name="text">
                <string>По самому низкому</string>
            </property>
        </action>
        <action name="action_binding_mode">
            <property name="text">
                <string>Режим привязки</string>
            </property>
        </action>
        <action name="action_ruler">
            <property name="text">
                <string>Линейка</string>
            </property>
        </action>
        <action name="action_inspector">
            <property name="text">
                <string>Инспектор объектов</string>
            </property>
        </action>
        <action name="action_connect_to_database">
            <property name="text">
                <string>Подключить к базе данных</string>
            </property>
        </action>
        <action name="action_objects_repository">
            <property name="text">
                <string>Репозиторий объектов</string>
            </property>
        </action>
        <action name="action_statistics">
            <property name="text">
                <string>Статистика</string>
            </property>
        </action>
        <action name="action_set_scheme">
            <property name="text">
                <string>Настройка схемы</string>
            </property>
        </action>
        <action name="action_clean_scheme">
            <property name="text">
                <string>Очистка схемы</string>
            </property>
        </action>
        <action name="action_relocate">
            <property name="text">
                <string>Перенести...</string>
            </property>
        </action>
        <action name="action_offset_transition_numbers">
            <property name="text">
                <string>Смещение номеров переходов</string>
            </property>
        </action>
        <action name="action_check_animation_transitions">
            <property name="text">
                <string>Проверка привязки анимации</string>
            </property>
        </action>
        <action name="action_copy_as_picture">
            <property name="text">
                <string>Копировать выделение как картинку</string>
            </property>
        </action>
        <action name="action_set_info_cedp">
            <property name="text">
                <string>Сформировать информацию о круге для ЦЭДП</string>
            </property>
        </action>
        <action name="action_give_info_cedp">
            <property name="text">
                <string>Передать информацию о круге в ЦЭДП</string>
            </property>
        </action>
        <action name="action_set_objects">
            <property name="text">
                <string>Настройка объектов</string>
            </property>
        </action>
        <action name="action_set_editor">
            <property name="text">
                <string>Настройка редактора</string>
            </property>
        </action>
        <action name="action_reference">
            <property name="text">
                <string>Справка</string>
            </property>
        </action>
        <action name="action_about_program">
            <property name="text">
                <string>О программе</string>
            </property>
        </action>
        <action name="action1_2">
            <property name="text">
                <string>1</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
